test/test_elb_lookup.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+import os
+this_directory = os.path.dirname(os.path.realpath(__file__))
+script_dir = os.path.realpath(os.path.join(this_directory,'../files'))
+import sys
+sys.path.append(script_dir)
+from unittest.mock import MagicMock
```

Comment:
MagicMock (commonly known as 'mocks') is a useful tool for unit testing when you don't want to actually use the external dependencies. In this case, we want to unit test something that calls out to AWS. Instead, we can provide sample return values using mocks. This allows us to test our logic without actually interacting with AWS.